### ISSUE TITLE
Assumes char is signed

### ISSUE THREAD

**Issue by arpruss** [sshipway/wanderer#14]
in main(), the getopt() return value is assigned to char c, and then compared against -1. This fails on systems (like my Raspberry PI) where char is unsigned. Instead, c should be declared int.

**Comment by sshipway** [comment]
yes, good point.  This used to work back 40 years ago when I first wrote it!  Care to submit a PR to fix?

**Comment by arpruss** [comment]
I had really enjoyed this about 35 years ago, and so I downloaded it and compiled it today. Worked great on Windows/Cygwin, but then got this hiccup.

I'll see if I can figure out how to do a PR. I haven't yet done any PRs since github disabled svn access.
